presto-kafka/src/test/java/io/prestosql/plugin/kafka/encoder/json/TestJsonEncoder.java

```diff
@@ -64,8 +88,36 @@ public void testColumnValidation()
         singleColumnEncoder(createVarcharType(20));
         singleColumnEncoder(createUnboundedVarcharType());
 
+        singleColumnEncoder(TIMESTAMP, "rfc2822", "");
+        singleColumnEncoder(TIMESTAMP_WITH_TIME_ZONE, "rfc2822", "");
+
+        for (String dataFormat : ImmutableList.of("custom-date-time", "iso8601")) {
```

Comment:
why passing format string for iso8601?

And more general question. What is purpose of this test at all? (I think I have missed it during previous reivews).
Is it about compatibility of different encoders and data types?
Maybe instead just calling out to `singleColumnEncoder` it would be more readable to wrap it into asserting method like you do for `assertUnsupportedColumnTypeException`. Even if assertion would be just "does not throw exception".
sth like
`assertCanCreateEncoder(() -> singleColumnEncoder(BIGINT);`